<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-EJSMVPFR" failures="1" tests="7" name="BankingApp Test" time="309.474" errors="0" timestamp="2022-10-03T10:10:13 IST">
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithValidCredentials" time="2.377"/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithInvalidCredentials" time="0.531"/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithValidUsernameAndInvalidPassword" time="0.497"/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithInvalidUsernameAndValidPassword" time="0.482"/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="maxlogin" time="0.497"/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginAfterMaxLogin" time="300.725">
    <failure type="java.lang.AssertionError" message="expected [http://gcreddy.com/project/admin/index.php123] but found [http://gcreddy.com/project/admin/index.php]">
      <![CDATA[java.lang.AssertionError: expected [http://gcreddy.com/project/admin/index.php123] but found [http://gcreddy.com/project/admin/index.php]
at com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest.loginAfterMaxLogin(TC1001_Administrator_loginTest.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginAfterMaxLogin -->
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="redirectingToCustomerTest" time="2.063"/>
</testsuite> <!-- BankingApp Test -->
